src/client/Client.cc

```diff
@@ -4141,8 +4140,11 @@ void Client::trim_caps(MetaSession *s, int max)
       }
     }
   }
-  ldout(cct, 20) << " clearing anchored inodes" << dendl;
-  anchor.clear();
+  ldout(cct, 20) << " trimming queued dentries: " << dendl;
+  for (const auto &dn : to_trim) {
+    trim_dentry(dn);
+  }
+  to_trim.clear();
 
   caps_size = s->caps.size();
   if (caps_size > max)
```

Comment:
this line still causes warning. how about change the 'max' parameter to uint64_t

Reply:
Thanks, fixed!